namespace/digest.go

```diff
@@ -6,9 +6,9 @@ import (
 )
 
 type IntervalDigest struct {
-	min    ID
-	max    ID
-	digest []byte
+	Min    ID     `json:"min"`
+	Max    ID     `json:"max"`
+	Digest []byte `json:"digest"`
 }
```

Comment:
I'm wondering if we should not get rid of this struct entirely and instead store this as raw bytes min||max||digest instead. For instance, when using this with ipfs, the CID will essentially be this plus some overhead bytes that stem from the multihash standard. 
We can merge this as is to unblock https://github.com/lazyledger/lazyledger-core/pull/312 but we should track this somewhere.